src/main/java/org/mitre/synthea/modules/LifecycleModule.java

```diff
@@ -409,6 +409,7 @@ private static boolean age(Person person, long time) {
       case 30:
         // "overeducated" -> suffix
         if ((person.attributes.get(Person.NAME_SUFFIX) == null)
+            && (person.attributes.get(Person.EDUCATION_LEVEL) != null)
```

Comment:
Were you finding people with `null` education level?

Reply:
Yes, IIRC I had to add that to avoid an NPE in the test suite.

Reply:
I removed this line and the test suite now passes - maybe something was fixed in the rebase.